card/mod/carrierwave/spec/set/type/file_spec.rb

```diff
@@ -383,7 +383,7 @@ def source_view card
       after { Cardio.config.file_storage = :local }
       it "copies file to local file system" do
         # not yet supported
-        expect { subject.update_attributes!(storage_type: :local) }
+        expect { Card[subject.name].update_attributes!(storage_type: :local) }
```

Comment:
The `subject` call creates the card and the `@file_changed` attrbute doesn't get reset. That's why I need a new object.